ores/wsgi/templates/home.html

```diff
@@ -1,51 +1,63 @@
 <html>
 <head>
-  <title>Objective Revision Evaluation Service</title>
-  <link rel="stylesheet" href="wikimedia-ui-static/MW/mediawiki.min.css">
-  <meta name="description" content="The Objective Revision Evaluation Service (ORES) is a web service that provides machine learning as a service for Wikimedia Projects"/>
-  <meta property="og:image" content="https://ores.wikimedia.org/static/images/ores_logo.svg"/>
+	<title>Objective Revision Evaluation Service</title>
+	<link rel="stylesheet" href="https://design.wikimedia.org/css/build/wmui-style-guide.min.css">
+	<link rel="stylesheet" href="/static/css/ores-site-specific.css">
+	<meta name="description" content="The Objective Revision Evaluation Service (ORES) is a web service that provides machine learning as a service for Wikimedia Projects"/>
+	<meta property="og:image" content="https://ores.wikimedia.org/static/images/ores_logo.svg"/>
+	<style type="text/css">
+		.notice {
+			border: 1px solid #b32424;
+			margin: 4px 2em;
+			background: #fee7e6;
+			padding: 0.25em 0.9em;
+		}
+		.site__logo {
+			-moz-background-size: 100%;
+			-webkit-background-size: 100% 100%;
+			background-image: url(/static/images/ores_logo.svg);
+			background-size: 100%;
```

Comment:
I think we need a "norepeat", cos I'm seeing a sliver of the green gear repeat at the bottom of the box.